library/common/extensions/retry/options/interface_binding/predicate.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include "envoy/upstream/retry.h"
+
+#include "library/common/extensions/retry/options/interface_binding/predicate.pb.h"
+#include "library/common/extensions/retry/options/interface_binding/predicate.pb.validate.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Retry {
+namespace Options {
+
+class InterfaceBindingRetryOptionsPredicate : public Upstream::RetryOptionsPredicate {
```

Comment:
Suggest we `s/InterfaceBinding/NetworkConfiguration/g`, as that's the namespacing I've landed on using elsewhere.